tests/integration/targets/backup_vault/tasks/main.yml

```diff
@@ -0,0 +1,55 @@
+---
+- module_defaults:
+    group/aws:
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      security_token: "{{ security_token | default(omit) }}"
+      region: "{{ aws_region }}"
+  block:
+    - name: create a key
+      kms_key:
+        alias: '{{ kms_key_alias }}'
+        state: present
+        enabled: yes
+        enable_key_rotation: no
```

Comment:
Can you also please extend the integration tests  with idempotency checks, check_mode, etc ?